armory/baseline_models/pytorch/mnist.py

```diff
@@ -46,6 +46,10 @@ def make_mnist_model(**kwargs):
 
 def get_art_model(model_kwargs, wrapper_kwargs, weights_file=None):
     model = make_mnist_model(**model_kwargs)
+
+    if torch.cuda.is_available():
+        model.cuda()
```

Comment:
`model.cuda()` is an in place operation? I.e., we don't need to do `model = model.cuda()` ?

Reply:
Yup it's inplace:
https://discuss.pytorch.org/t/solved-make-sure-that-pytorch-using-gpu-to-compute/4870/2